### DISCUSSION TITLE
Spawning as seagull on first mission load

### DISCUSSION THREAD

**Discussion by Morbakos** [Vdauphin/HeartsAndMinds#1511] [Question and answer]
Hi !

I have a problem with the mission (version 1.23.3) after switching to the Esseker map.
When the mission first loads, players spawn as seagulls, and an administrator is forced to manually restart the mission so that it loads correctly. 

Here's an extract of my .rpt: https://gist.github.com/Morbakos/6bf125b233afe8e0b2799ac15836c251

Any idea why this happens and how to fix it?

**Comment by Vdauphin** [comment]
Hello,

> Any idea why this happens and how to fix it?

First fix all errors appearing in your .rpt and then if the issue persists, remove one by one mods until it works again

Cheers

**Reply by Morbakos** [reply]
Thing is, I don't have the issue when I start the mission from the editor, only on dedicated server

**Reply by Vdauphin** [reply]
For H&M editor doesn't matter as H&M only works on dedicated server
Depend on misssion/mods, code execution in editor and dedicated server can be way different due to locality. Expecting something working on editor do not guarantee it will work on dedicated server: https://community.bistudio.com/wiki/Multiplayer_Scripting
If your mission works on editor and not on dedicated them you need to change the code to support dedicated server

But again, not fixing errors generating 15000 error lines in your .rpt in less than 8 second is not a way to go.

Btw .rpt report H&M version 1.23.2

**Comment by Souza-R** [comment]
I had a problem with this a while back. It was only happening on a specific map... I don't remember which one. But the solution I found was to reduce the number of vehicles in the mission. First, I removed them all, and the spawn happened normally. Then, I started adding them back until the seagull devil appeared, and that's when I knew the limit.
